src/Translatable/TranslatableLocales.php

```diff
@@ -0,0 +1,76 @@
+<?php
+namespace Dimsav\Translatable;
+
+
+class TranslatableLocales
```

Comment:
I'm wondering if we rename this facade to just `Translatable` since we might add other stuff in the future not related explicitly to locales. What do you thing?